schema.dbml

```diff
@@ -600,39 +600,30 @@ Table invitro_selection_results as Vitro {
   }
 }
 
-Table invivo_selection_results as Vivo {
-  // TODO: refactor to use subjects table
+Enum antibody_dosage_unit_enum {
+  "mg/kg"
+}
+
+Table rx_invivo as RxVivo {
   ref_name varchar
   rx_name varchar
-  dosage varchar
-  host varchar
-  infection_name varchar [ref: > I.iso_name]
-  gene varchar [
-    note: 'Mutation gene'
-  ]
-  position int [
-    note: 'Position of this mutation'
-  ]
-  amino_acid amino_acid_enum [
-    note: 'Amino acid of this mutation'
-  ]
-  subject varchar
-  sampling varchar
-  num_subjects int
-  num_subjects_with_mut int
```

Comment:
`num_subjects_with_mut` is removed from the core table. The value is now saved at `subjects.num_subjects`.